### ISSUE TITLE
[close]KeyError: 'fort_id'

### ISSUE THREAD

**Issue by Zgrkaralar** [PokemonGoF/PokemonGo-Bot#1067]
### KeyError: 'fort_id'
After #1039 change i update my dev version now my problem is that .

<img width="552" alt="screen shot 2016-07-27 at 12 48 52 am" src="https://cloud.githubusercontent.com/assets/879674/17157380/89bf035e-5397-11e6-83fe-950a2a225ceb.png">

### Other Information
OS: Ubuntu
Git Commit: 9c99cc0de7e5b0c35762cae9b657c495d32e3ade

**Comment by und3rdg** [comment]
same here 
`

[23:14:59] Starting PokemonGo Bot....
[23:14:59] Scanning area for objects....
[23:14:59] Walking from (51.500575, -0.127703) to [51.500575, -0.127703] 0.00km
[████████████████████████████████████████] 100%
[23:15:01] Attempting to evolve all pokemons ...
[23:15:04] Something rustles nearby!
Traceback (most recent call last):
  File "./pokecli.py", line 246, in <module>
    main()
  File "./pokecli.py", line 237, in main
    bot.take_step()
  File "/home/undg/pokemon/bot/PokemonGo-Bot/pokemongo_bot/__init__.py", line 46, in take_step
    self.work_on_cell(cell, location)
  File "/home/undg/pokemon/bot/PokemonGo-Bot/pokemongo_bot/__init__.py", line 189, in work_on_cell
    if self.catch_pokemon(pokemon) == PokemonCatchWorker.NO_POKEBALLS:
  File "/home/undg/pokemon/bot/PokemonGo-Bot/pokemongo_bot/__init__.py", line 337, in catch_pokemon
    return_value = worker.work()
  File "/home/undg/pokemon/bot/PokemonGo-Bot/pokemongo_bot/cell_workers/pokemon_catch_worker.py", line 31, in work
    response_dict = self.create_encounter_api_call()
  File "/home/undg/pokemon/bot/PokemonGo-Bot/pokemongo_bot/cell_workers/pokemon_catch_worker.py", line 400, in create_encounter_api_call
    fort_id = self.pokemon['fort_id']
KeyError: 'fort_id'
`

**Comment by Zgrkaralar** [comment]
I think who change it on other Issues #1045 #1046 maybe have same problem

**Comment by aslafy-z** [comment]
Commit just been reverted, please repull. #1065

**Comment by Zgrkaralar** [comment]
@aslafy-z Thank you when i change to head 1056 working ;)

**Comment by Zgrkaralar** [comment]
@aslafy-z :( after visit pokestore now this problem :(


<img width="663" alt="screen shot 2016-07-27 at 1 31 59 am" src="https://cloud.githubusercontent.com/assets/879674/17157804/90cd64e4-539a-11e6-8f49-8bba7f6cd1e2.png">

**Comment by aslafy-z** [comment]
@Zgrkaralar This is another isue due to your outdated config file. Recreate it from configs/config.json.example

**Comment by Zgrkaralar** [comment]
<img width="754" alt="screen shot 2016-07-27 at 2 03 48 am" src="https://cloud.githubusercontent.com/assets/879674/17158363/67718ca2-539e-11e6-928f-a876be8f6261.png">
 Now problem is this

**Comment by aslafy-z** [comment]
Please open another issue, and someone close this one please.

**Comment by surfaace** [comment]
Commit has been reverted, please repull. #1065
